package.json

```diff
@@ -46,14 +41,14 @@
         "eslint": "7.20.0",
         "eslint-config-airbnb-base": "14.2.1",
         "eslint-plugin-import": "2.22.1",
-        "eslint-config-prettier": "7.0.0",
+        "eslint-config-prettier": "8.0.0",
         "fastify": "3.12.0",
         "postcss": "8.2.6",
-        "rollup": "2.39.0",
         "semantic-release": "17.3.9",
         "tap": "14.11.0"
     },
     "dependencies": {
+        "@eik/common": "4.0.0-next.4",
```

Comment:
Wondering if using this version is going to cause any issues? Maybe safer to go with v2 until v4 is released?

Reply:
Yeah. I am not 100% sure how to fix this one. Semantic Release did just bump two major version without any good reason. Even reported that as a bug: https://github.com/semantic-release/semantic-release/issues/1809

Any good ideas on how to solve it?

Reply:
Perhaps roll the version being used here back to v2 for now?

If you want to fix release of v4 instead of v3 could you manually change the version to v3.0.0-next.1 and then update the dist tag? When releasing v4.0.0-next one would need to manually jump over to 4.0.0-next.5 I assume. Might not be worth it to fix it.